safe/definitions/provenance.py

```diff
@@ -195,6 +195,38 @@
     'provenance_key': layer_purpose_profiling['key']
 }
 
+# Layers ID
+provenance_layer_exposure_summary_id = {
+    'key': 'provenance_layer_exposure_summary_id',
+    'name': layer_purpose_exposure_summary['name'] + ' ID',
```

Comment:
Should we create a method for this? Just to make sure we have the same behaviour.